packages/components/data-table/src/cell.styles.js

```diff
@@ -148,6 +148,30 @@ const getSortableHeaderStyles = (props) => css`
     }
   }
 `;
+
```

Comment:
Just moved this whole block up so that these components can be used as selectors below.